resources/lang/en.json

```
@@ -972,16 +979,13 @@
   "Warnings": "Warnings",
   "no warnings to report": "No Warnings To Report",
   "Assignment Rules": "Assignment Rules",
-  "Enable Interstitial": "Enable Interstitial",
   "redirected to my next assigned task": "redirected to my next assigned task",
```

This was replaced with: Directs Task assignee to the next assigned Task
```suggestion
  "Directs Task assignee to the next assigned Task": "Directs Task assignee to the next assigned Task",
```